packages/framework/store/src/store/doc/query.ts

```diff
@@ -0,0 +1,82 @@
+import isMatch from 'lodash.ismatch';
+
+import type { BlockModel } from '../../schema/index.js';
+import type { Block } from './block/index.js';
+
+import { BlockViewType } from './doc.js';
+
+export type QueryMatch = {
+  id?: string;
+  flavour?: string;
+  props?: Record<string, unknown>;
+  viewType: BlockViewType;
```

Comment:
just wonder why we need to define viewtype in query. If there are only include and not_include, the design here will be simpler.

Reply:
by pass is needed for render a block without render the view of its ancestors